Examples npm run dev failing

I've been having troubles running "npm run build" from examples/gh-pages

npm-debug.log outputs the following:

```
0 info it worked if it ends with ok
1 verbose cli [ 'node', '/Users/lukepint/.node/bin/npm', 'run', 'dev' ]
2 info using npm@2.8.4
3 info using node@v0.12.0
4 verbose node symlink /usr/local/bin/node
5 verbose run-script [ 'predev', 'dev', 'postdev' ]
6 info predev gh-pages@1.0.0
7 info dev gh-pages@1.0.0
8 verbose unsafe-perm in lifecycle true
9 info gh-pages@1.0.0 Failed to exec dev script
10 verbose stack Error: gh-pages@1.0.0 dev: `npm run clean && webpack-dev-server --content-base ../../public`
10 verbose stack Exit status 1
10 verbose stack     at EventEmitter.<anonymous> (/Users/lukepint/.node/lib/node_modules/npm/lib/utils/lifecycle.js:213:16)
10 verbose stack     at EventEmitter.emit (events.js:110:17)
10 verbose stack     at ChildProcess.<anonymous> (/Users/lukepint/.node/lib/node_modules/npm/lib/utils/spawn.js:24:14)
10 verbose stack     at ChildProcess.emit (events.js:110:17)
10 verbose stack     at maybeClose (child_process.js:1008:16)
10 verbose stack     at Process.ChildProcess._handle.onexit (child_process.js:1080:5)
11 verbose pkgid gh-pages@1.0.0
12 verbose cwd /Users/lukepint/workspace/boomplans/react-google-maps-master/examples/gh-pages
13 error Darwin 14.0.0
14 error argv "node" "/Users/lukepint/.node/bin/npm" "run" "dev"
15 error node v0.12.0
16 error npm  v2.8.4
17 error code ELIFECYCLE
18 error gh-pages@1.0.0 dev: `npm run clean && webpack-dev-server --content-base ../../public`
18 error Exit status 1
19 error Failed at the gh-pages@1.0.0 dev script 'npm run clean && webpack-dev-server --content-base ../../public'.
19 error This is most likely a problem with the gh-pages package,
19 error not with npm itself.
19 error Tell the author that this fails on your system:
19 error     npm run clean && webpack-dev-server --content-base ../../public
19 error You can get their info via:
19 error     npm owner ls gh-pages
19 error There is likely additional logging output above.
20 verbose exit [ 1, true ]
```

In addition, I found this in the output of the run:

```
{ file: '/Users/lukepint/workspace/boomplans/examples/gh-pages/node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/_grid.scss' }
/Users/lukepint/workspace/boomplans/examples/gh-pages/node_modules/webpack-dev-server/node_modules/webpack-dev-middleware/middleware.js:85
			if(err) throw err;
			              ^
TypeError: tmp/server.js
--------------------------------------------------------------------------------options must be an object
    at TypeError (native)
    at Object.exports.createScript (vm.js:44:10)
    at module.exports (/Users/lukepint/workspace/boomplans/examples/gh-pages/node_modules/eval/eval.js:55:8)
    at IsomorphicReactPluginFactory.evaluate (/Users/lukepint/workspace/boomplans/examples/gh-pages/IsomorphicReactPluginFactory.js:56:12)
    at IsomorphicReactPluginFactory.runCompilation (/Users/lukepint/workspace/boomplans/examples/gh-pages/IsomorphicReactPluginFactory.js:68:32)
    at IsomorphicReactPluginFactory.receivedClientAssets (/Users/lukepint/workspace/boomplans/examples/gh-pages/IsomorphicReactPluginFactory.js:33:10)
    at ClientPlugin.<anonymous> (/Users/lukepint/workspace/boomplans/examples/gh-pages/IsomorphicReactPluginFactory.js:97:24)
    at Tapable.applyPluginsAsync (/Users/lukepint/workspace/boomplans/examples/gh-pages/node_modules/webpack/node_modules/tapable/lib/Tapable.js:71:13)
    at Tapable.Compiler.emitAssets (/Users/lukepint/workspace/boomplans/examples/gh-pages/node_modules/webpack/lib/Compiler.js:216:7)
    at Watching.<anonymous> (/Users/lukepint/workspace/boomplans/examples/gh-pages/node_modules/webpack/lib/Compiler.js:45:18)
--------------------------------------------------------------------------------
```

Can you provide any insight into why this might be failing?

Thanks

Likewise seeing this error when building the example pages.

```
npm ERR! code ELIFECYCLE
npm ERR! gh-pages@1.0.0 build_client: npm run clean && webpack -p
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the gh-pages@1.0.0 build_client script 'npm run clean && webpack -p'.
npm ERR! This is most likely a problem with the gh-pages package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR! npm run clean && webpack -p
```

(This is on a fresh clone/install)

I'd like to contribute to this project with however I'm running into the following error even in  `node@0.10` when running `npm run dev`.

```
ERROR in ./~/react-toastr/lib/ToastMessage/jQueryMixin.js
Module not found: Error: Cannot resolve module 'jquery' in
```

Any quick solutions to fix this or remove the dependency?

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
+- '0.12'
 - '0.10'
 env:
   global:

--- a/examples/gh-pages/IsomorphicReactPluginFactory.js
+++ b/examples/gh-pages/IsomorphicReactPluginFactory.js
@@ -1,7 +1,7 @@
 "use strict";
 
 var React = require("react");
-var evaluate = require("eval");
+var evaluate = require("eval-as-module");
 
 var SEPERATOR = "----------------------------------------";// 40 chars
 
@@ -53,7 +53,7 @@ IsomorphicReactPluginFactory.prototype.evaluate = function (serverPath, errorMes
   var includeGlobals = this.evaluateOptions.includeGlobals;
 
   try {
-    return evaluate(source, includeGlobals);
+    return evaluate(source, includeGlobals).exports;
   } catch (error) {
     error.message = serverPath + "\n" + SEPERATOR + SEPERATOR + error.message;
     error.stack = error.stack + "\n" + SEPERATOR + SEPERATOR + source + "\n" + SEPERATOR + SEPERATOR;

--- a/examples/gh-pages/package.json
+++ b/examples/gh-pages/package.json
@@ -4,30 +4,32 @@
   "scripts": {
     "clean": "rm -f ../../public/index.html && rm -rf ../../public/assets && rm -rf ../../public/tmp",
     "build_client": "npm run clean && webpack -p",
+    "build": "npm run clean && webpack -d",
     "dev": "npm run clean && webpack-dev-server -d --content-base ../../public"
   },
   "devDependencies": {
-    "babel-core": "^5.1.13",
+    "babel-core": "^5.3.3",
     "babel-loader": "^5.0.0",
-    "bootstrap-sass": "^3.3.4",
-    "classnames": "^1.2.1",
     "css-loader": "^0.12.0",
-    "eval": "^0.1.0",
-    "extract-text-webpack-plugin": "^0.7.0",
+    "eval-as-module": "^0.1.1",
+    "extract-text-webpack-plugin": "^0.8.0",
     "file-loader": "^0.8.1",
     "github-fork-ribbon-css": "git+https://github.com/tomchentw/github-fork-ribbon-css.git#gh-pages",
-    "html-webpack-plugin": "^1.1.0",
-    "jquery": "^2.1.1",
-    "node-sass": "^2.1.1",
-    "prismjs": "git+https://github.com/PrismJS/prism.git#master",
+    "node-sass": "^3.1.0",
     "raw-loader": "^0.5.1",
-    "react-hot-loader": "^1.2.5",
+    "react-hot-loader": "^1.2.7",
+    "sass-loader": "^1.0.1",
+    "style-loader": "^0.12.2",
+    "webpack": "^1.9.5",
+    "webpack-dev-server": "^1.8.2"
+  },
+  "dependencies": {
+    "bootstrap-sass": "^3.3.4",
+    "classnames": "^2.1.1",
+    "prismjs": "git+https://github.com/PrismJS/prism.git#master",
+    "react": "^0.13.3",
     "react-prism": "^1.3.0",
-    "react-toastr": "^1.2.0",
-    "sass-loader": "^0.6.0",
-    "style-loader": "^0.12.1",
-    "toastr": "^2.0.4",
-    "webpack": "^1.4.7",
-    "webpack-dev-server": "^1.6.5"
+    "react-toastr": "^1.3.0",
+    "toastr": "^2.0.4"
   }
 }

--- a/examples/gh-pages/package.json
+++ b/examples/gh-pages/package.json
@@ -24,6 +24,7 @@
     "webpack-dev-server": "^1.8.2"
   },
   "dependencies": {
+    "animate.css": "^3.2.5",
     "bootstrap-sass": "^3.3.4",
     "classnames": "^2.1.1",
     "prismjs": "git+https://github.com/PrismJS/prism.git#master",

--- a/examples/gh-pages/scripts/ReactRoot.js
+++ b/examples/gh-pages/scripts/ReactRoot.js
@@ -194,7 +194,7 @@ class ReactRoot extends React.Component {
           </div>
           <ToastContainer 
             ref="toast"
-            toastMessageFactory={React.createFactory(ToastMessage.jQuery)}/>
+            toastMessageFactory={React.createFactory(ToastMessage.animation)}/>
 
           <ComponentPlayground
             className="row row--full-height"

--- a/examples/gh-pages/styles/_bootstrap_override.scss
+++ b/examples/gh-pages/styles/_bootstrap_override.scss
@@ -1,50 +1,50 @@
 // Core variables and mixins
-@import "bootstrap/variables";
-@import "bootstrap/mixins";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/variables";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/mixins";
 
 // Reset and dependencies
-@import "bootstrap/normalize";
-@import "bootstrap/print";
-@import "bootstrap/glyphicons";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/normalize";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/print";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/glyphicons";
 
 // Core CSS
-@import "bootstrap/scaffolding";
-@import "bootstrap/type";
-@import "bootstrap/code";
-@import "bootstrap/grid";
-@import "bootstrap/tables";
-@import "bootstrap/forms";
-@import "bootstrap/buttons";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/scaffolding";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/type";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/code";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/grid";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/tables";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/forms";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/buttons";
 
 // Components
-@import "bootstrap/component-animations";
-@import "bootstrap/dropdowns";
-@import "bootstrap/button-groups";
-@import "bootstrap/input-groups";
-@import "bootstrap/navs";
-@import "bootstrap/navbar";
-@import "bootstrap/breadcrumbs";
-@import "bootstrap/pagination";
-@import "bootstrap/pager";
-@import "bootstrap/labels";
-@import "bootstrap/badges";
-@import "bootstrap/jumbotron";
-@import "bootstrap/thumbnails";
-@import "bootstrap/alerts";
-@import "bootstrap/progress-bars";
-@import "bootstrap/media";
-@import "bootstrap/list-group";
-@import "bootstrap/panels";
-@import "bootstrap/responsive-embed";
-@import "bootstrap/wells";
-@import "bootstrap/close";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/component-animations";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/dropdowns";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/button-groups";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/input-groups";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/navs";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/navbar";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/breadcrumbs";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/pagination";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/pager";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/labels";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/badges";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/jumbotron";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/thumbnails";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/alerts";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/progress-bars";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/media";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/list-group";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/panels";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/responsive-embed";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/wells";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/close";
 
 // Components w/ JavaScript
-@import "bootstrap/modals";
-@import "bootstrap/tooltip";
-@import "bootstrap/popovers";
-@import "bootstrap/carousel";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/modals";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/tooltip";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/popovers";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/carousel";
 
 // Utility classes
-@import "bootstrap/utilities";
-@import "bootstrap/responsive-utilities";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/utilities";
+@import "~bootstrap-sass/assets/stylesheets/bootstrap/responsive-utilities";

--- a/examples/gh-pages/styles/_bootstrap_variables.scss
+++ b/examples/gh-pages/styles/_bootstrap_variables.scss
@@ -1,1 +1,4 @@
-$icon-font-path: "//maxcdn.bootstrapcdn.com/bootstrap/3.2.0";
+$icon-font-path: "//maxcdn.bootstrapcdn.com/bootstrap/3.3.4";
+
+// Adjust this to enjoy Hot Module Replacement :)
+// $font-size-base: 20px;

--- a/examples/gh-pages/styles/index.scss
+++ b/examples/gh-pages/styles/index.scss
@@ -1,8 +1,8 @@
-@import url(//fonts.googleapis.com/css?family=Lato:400,900);
+@import url("//fonts.googleapis.com/css?family=Lato:400,900");
 
-@import url(~github-fork-ribbon-css/gh-fork-ribbon.css);
-@import url(~prismjs/themes/prism.css);
-@import url(~toastr/toastr.min.css);
+@import url("~github-fork-ribbon-css/gh-fork-ribbon.css");
+@import url("~prismjs/themes/prism.css");
+@import url("~toastr/toastr.min.css");
 
 @import "./bootstrap_variables";
 @import "./bootstrap_override";

--- a/examples/gh-pages/styles/index.scss
+++ b/examples/gh-pages/styles/index.scss
@@ -2,7 +2,8 @@
 
 @import url("~github-fork-ribbon-css/gh-fork-ribbon.css");
 @import url("~prismjs/themes/prism.css");
-@import url("~toastr/toastr.min.css");
+@import url("~animate.css/animate.css");
+@import url("~toastr/toastr.scss");
 
 @import "./bootstrap_variables";
 @import "./bootstrap_override";

--- a/examples/gh-pages/webpack.config.js
+++ b/examples/gh-pages/webpack.config.js
@@ -55,10 +55,6 @@ clientConfig = {
     ],
   },
   plugins: [
-    new webpack.ProvidePlugin({
-      $: "jquery",
-      jQuery: "jquery",
-    }),
     new ExtractTextPlugin("[name].css", {
       disable: IS_DEVELOPMENT,
     }),

--- a/examples/gh-pages/webpack.config.js
+++ b/examples/gh-pages/webpack.config.js
@@ -18,11 +18,7 @@ var Path = require("path"),
     BABEL_LOADER = "babel-loader?stage=1",
     STYLE_LOADER = "style-loader",
     CSS_LOADER = "css-loader?root=../",
-    SASS_LOADER = CSS_LOADER + "!sass-loader?" + JSON.stringify({
-      includePaths: [
-        Path.resolve(__dirname, "./node_modules/bootstrap-sass/assets/stylesheets"),
-      ]
-    });
+    SASS_LOADER = CSS_LOADER + "!sass-loader";
 
 isomorphicReactPlugin = new IsomorphicReactPluginFactory({
   serverComponentPath: "tmp/server.js",

